app/js/modules/pianoroll/notes.js

```diff
@@ -9,18 +9,13 @@ define(function (require) {
     Notes.oktava=[Notes.names.C,Notes.names.D,Notes.names.E,Notes.names.F,Notes.names.G,Notes.names.A,Notes.names.B];
     Notes.pianoroll=[[Notes.names.A,Notes.names.B],Notes.oktava,Notes.oktava,Notes.oktava,Notes.oktava,Notes.oktava,Notes.oktava,Notes.oktava,[Notes.names.C]];
     Notes.checkSharp=function(note){
-        if(note==Notes.names.F || note==Notes.names.C){
-            return false
-        }
-        else{
-            return true
-        }
+        return note != Notes.names.F && note != Notes.names.C;
```

Comment:
BTW, why **type-converting** comparison (==) was used in preference of **strict** (===)?